src/main/java/com/google/api/gax/core/DefaultNanoClock.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2016, Google Inc. All rights reserved.
```

Comment:
This was not created but moved to another package, so I kept the date as is. Should I update it for moved files too?